src/languages/es.ts

```diff
@@ -1389,6 +1389,37 @@ export default {
         letsDoubleCheck: 'Revisemos que todo esté bien',
         byAddingThisBankAccount: 'Agregando esta cuenta bancaria, confirmas que as leído, entendido y aceptado',
     },
+    businessInfoStep: {
+        businessInfo: 'Información de Negocio',
+        enterTheNameOfYourBusiness: 'Introduzca el nombre de su negocio.',
+        businessName: 'Nombre del Negocio',
+        enterYourCompanysTaxIdNumber: 'Introduzca el número de identificación fiscal.',
+        taxIDNumber: 'Número de identificación fiscal',
+        taxIDNumberPlaceholder: '9 dígitos',
+        enterYourCompanysWebsite: 'Introduzca el sitio web de su compañia.',
+        companyWebsite: 'Sitio web de la compañia',
+        enterYourCompanysPhoneNumber: 'Introduzca el número de teléfono de su compañia.',
+        enterYourCompanysAddress: 'Introduzca el la dirección de su compañia.',
+        selectYourCompanysType: 'Seleccione el tipo de compañia.',
+        companyType: 'Tipo de compañia',
+        incorporationType: {
+            LLC: 'SRL',
+            CORPORATION: 'Corporación',
+            PARTNERSHIP: 'Asociación',
+            COOPERATIVE: 'Cooperativa',
+            SOLE_PROPRIETORSHIP: 'Empresa unipersonal',
+            OTHER: 'Otros',
+        },
+        selectYourCompanysIncorporationDate: 'Seleccione la fecha de constitución de su empresa.',
+        incorporationDate: 'Fecha de constitución',
+        incorporationDatePlaceholder: 'Fecha de inicio (yyyy-mm-dd)',
+        incorporationState: 'Estado de constitución',
+        pleaseSelectTheStateYourCompanyWasIncorporatedIn: 'Seleccione el estado en el que se constituyó su empresa.',
+        letsDoubleCheck: 'Verifiquemos que todo esté correcto',
+        companyAddress: 'Dirección de la empresa',
+        listOfRestrictedBusinesses: 'lista de negocios restringidos',
+        confirmCompanyIsNot: 'Confirmo que esta empresa no está en la',
+    },
```

Comment:
Can you confirm the verification of these in slack?

Reply:
as with the other parts of this flow, given time constraints, it's been said that we will have a final check towards the end of the work on it, accepted as non blocking by Vit [here](https://callstack-hq.slack.com/archives/C02MW39LT9N/p1699364922579169?thread_ts=1699364825.408549&cid=C02MW39LT9N)